docs/fastdds/logging/logging_configuration.rst

```diff
@@ -187,7 +187,7 @@ The logging module's configuration can be reset to default settings with member
     * Disabling :ref:`dds_layer_log_file_context` component.
     * Enabling :ref:`dds_layer_log_function_name` component.
     * Clear all :ref:`dds_layer_log_filter`.
-    * Clear all consumers and set a STDOUT consumer.
+    * Clear all consumers and set a STDOUT or STDOUTERR consumer (depending on the CMake option |LOG_CONSUMER_DEFAULT|).
```

Comment:
What about the threshold level? I'm guessing the new `STDOUTERR` will have the default value, right?

Reply:
Yes, the default consumer is reset.